stremio-core-protobuf/src/model/fields/addon_detail.rs

```diff
@@ -0,0 +1,44 @@
+use crate::bridge::{FromProtobuf, ToProtobuf};
+use crate::protobuf::stremio::core::models;
+use stremio_core::models::addon_details::{AddonDetails, Selected};
+use stremio_core::models::ctx::Ctx;
+
+impl FromProtobuf<Selected> for models::addon_details::Selected {
+    fn from_protobuf(&self) -> Selected {
+        let transport_url = self.transport_url.from_protobuf();
+
+        Selected {
+            transport_url: if transport_url.scheme() == "stremio" {
```

Comment:
Maybe this should be done in `core` itself since it seems relevant to all the platforms (ie. copy pasting addon install links)?

Reply:
Maybe it does make sense for the AddonDetail, I'm just worried that some replacements will be made in core-* crates and some in core. E.g. the deeplinks urls in core-web are replaced with `#` if we start sprinkling replacements here and there this would lead to even more confusion and hard maintenance.

I will look into this particular case though not a priority atm.
Created a task for this:

https://github.com/Stremio/stremio-tasks/issues/290